sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -83,12 +83,17 @@ public partial class DynamicJson : System.Dynamic.IDynamicMetaObjectProvider
         public override string ToString() { throw null; }
         public void WriteTo(System.Text.Json.Utf8JsonWriter writer) { }
     }
-    public partial class JsonObjectSerializer : Azure.Core.ObjectSerializer
+    public partial interface ISerializedMemberNameProvider
+    {
+        string? GetSerializedMemberName(System.Reflection.MemberInfo memberInfo);
+    }
+    public partial class JsonObjectSerializer : Azure.Core.ObjectSerializer, Azure.Core.ISerializedMemberNameProvider
     {
         public JsonObjectSerializer() { }
         public JsonObjectSerializer(System.Text.Json.JsonSerializerOptions options) { }
         public override object Deserialize(System.IO.Stream stream, System.Type returnType, System.Threading.CancellationToken cancellationToken) { throw null; }
         public override System.Threading.Tasks.ValueTask<object> DeserializeAsync(System.IO.Stream stream, System.Type returnType, System.Threading.CancellationToken cancellationToken) { throw null; }
+        public string? GetSerializedMemberName(System.Reflection.MemberInfo memberInfo) { throw null; }
```

Comment:
This should be implemented explicitly.

Reply:
I don't necessarily disagree, but why? Apart from taking a `MemberInfo` instead of a simple `string` name like Json.NET and STJ would (though, they do that on serialized type info, so they only need a string name), those implementations expose their name mapping which does come in handy - samples show people to use them, and I use them in our implementations. I don't see how effectively hiding these from the class itself benefits customers more than exposing them. Merely using the class won't requiring `using System.Reflection` - only if you call the method itself.

Reply:
Because this isn't something intended for regular customers to use.  A lot of people will end up looking at the `JsonObjectSerializer` type and I think implementing this explicitly will help them better understand how to use it.  It's not a big thing, but we've done this elsewhere like `SearchDocument` for the same reason.